### PR TITLE
Mock login & Authorization

### PR DESCRIPTION
**Pull request by nbittich**

### CODE REVIEW

File: config/resources/auth.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
+    "version": "0.1",
+    "prefixes": {
+        "foaf": "http://xmlns.com/foaf/0.1/",
+        "besluit": "http://data.vlaanderen.be/ns/besluit#",
+        "dct": "http://purl.org/dc/terms/",
+        "skos": "http://www.w3.org/2004/02/skos/core#"
+    },
+    "resources": {
+        "users": {
```
Comment (Windvis) [inline config/resources/auth.json:10]: This doesn't require a match config in the dispatcher?

Comment (nbittich) [reply config/resources/auth.json:10]: not for now (the mock-login doesn't require it) but might be needed if we do a specific page to show the user's information at some point.

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -4,9 +4,13 @@ services:
   identifier:
     ports:
       - "80:80"
-  db:
+  triplestore:
     ports:
       - "8890:8890"
+  mocklogin:
```
Comment (Windvis) [inline docker-compose.dev.yml:10]: This means we need to add it to the deployed apps docker.overrides.yml file, right? I guess that's how it is done in the other apps?

Comment (nbittich) [reply docker-compose.dev.yml:10]: yes indeed, it has to be added on the server where it will be deployed also

**Review by bdevloed — APPROVED** [review]

**Pull request merged**